workspaces/index.md

```diff
@@ -20,11 +20,15 @@ icon:
 description: "Learn about development workspaces in Coder."
 ---
 
-A workspace is your remote development environment. All workspaces are based on
-a shared [image](../images/index.md) and therefore reproducible.
+A workspace is your remote development environment.
 
 ![Workspaces](../assets/workspaces/workspaces.svg)
 
+All workspaces are based on a shared [image](../images/index.md) and therefore
+reproducible.
+
+![The anatomy of a workspace](../assets/workspaces/workspace-anatomy.png)
```

Comment:
@jmilling33 this image has a typo, `coder.taml`

Reply:
Just fixed!
![Copy of Anatomy of a Coder WorkspaceFinal (5)](https://user-images.githubusercontent.com/75645961/125690866-53186a21-41c6-4874-a75e-d2627f93ac7a.png)

Reply:
@jmilling33 Thanks! For the examples, I think our style is to use "e.g.," (with the comma) rather than "ex."

I don't think we need the parentheses in "Cloud IDEs"

Reply:
Gotcha. Thanks for the catch. 
![Copy of Anatomy of a Coder WorkspaceFinal (6)](https://user-images.githubusercontent.com/75645961/125691591-15e9f30b-c6ad-4b01-9f09-bb93b6069810.png)